packages/headless/src/commerce.index.ts

```diff
@@ -41,14 +41,8 @@ export type {
 } from './controllers/commerce/product-listing/headless-product-listing';
 export {buildProductListing} from './controllers/commerce/product-listing/headless-product-listing';
 
-export type {
-  PagerInitialState,
-  PagerOptions,
-  PagerProps,
-  Pager,
-  PagerState,
-} from './controllers/commerce/product-listing/pager/headless-product-listing-pager';
-export {buildPager} from './controllers/commerce/product-listing/pager/headless-product-listing-pager';
+export type {ProductListingPagination} from './controllers/commerce/product-listing/pagination/headless-product-listing-pagination';
```

Comment:
Why the rename from pager to pagination? Other pagination controllers use the term "pager", from my understanding.

Reply:
I'm still hesitating on the controller name, TBH, but here's my reasoning:

I decided to go with "pagination" because that's the name of the property in the product listing API.

Also, I made the name more specific (i.e., productListingPagination) essentially to help with search relevance. Since this pagination controller works completely differently from the pager controllers we have on other engines, I don't want people to be confused if it surfaces in search requests on docs.coveo.com, for instance.